eclair-core/src/test/scala/fr/acinq/eclair/TestConstants.scala

```diff
@@ -139,7 +139,7 @@ object TestConstants {
       initialRandomReconnectDelay = 5 seconds,
       maxReconnectInterval = 1 hour,
       chainHash = Block.RegtestGenesisBlock.hash,
-      channelFlags = 1,
+      channelFlags = ChannelFlags(announceChannel = true),
```

Comment:
nit: why not having a `ChannelFlags.Public` instance the way you defined a `ChannelFlags.Private`?

Reply:
I flipped-flopped on this. Those shortcuts woudn't make sense if we had more than one bit set.

Put it back in 8d8ee2f2f.